### ISSUE TITLE
Can't minimize docker desktop on ventura

### ISSUE THREAD

**Issue by fr-rose-steven** [docker/for-mac#6727]
- [x] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
Closing the window with the red x will close the window and an icon is displayed on the menu bar.
Opening docker desktop from somewhere like launchpad will reopen docker desktop.

### Actual behavior
The window is closed, docker is running in the background. There is no icon in the menu bar.
You have to kill all docker apps in activity monitor to be able to reopen docker desktop.

### Information
  - macOS Version: 13.2 (22D49)
  - Intel chip or Apple chip: M1
  - Docker Desktop Version: 4.16.2 (95914)

Seems same bug as https://github.com/docker/for-mac/issues/6440

### Output of `/Applications/Docker.app/Contents/MacOS/com.docker.diagnose check`
[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0013: is the $PATH ok?
[PASS] DD0003: is the Docker CLI working?
[PASS] DD0038: is the connection to Docker working?
[PASS] DD0014: are the backend processes running?
[PASS] DD0007: is the backend responding?
[PASS] DD0008: is the native API responding?
[PASS] DD0009: is the vpnkit API responding?
[PASS] DD0010: is the Docker API proxy responding?
[SKIP] DD0030: is the image access management authorized?
[PASS] DD0033: does the host have Internet access?
[PASS] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0032: do Docker networks overlap with host IPs?
No fatal errors detected.

### Steps to reproduce the behavior
  1. install docker desktop
  2. start docker desktop
  3. click red x to close window

**Comment by datta-aerosys** [comment]
Click on Docker icon on Menu bar, and click on `Dashboard`, that opens up the Docker Desktop for me.
Ventura 13.1
Docker Desktop 4.16.2

**Comment by fr-rose-steven** [comment]
@datta-aerosys ah thanks, but I don't even have an icon in the Menu bar

**Comment by datta-aerosys** [comment]
> @datta-aerosys ah thanks, but I don't even have an icon in the Menu bar

Ohh, ok. Is your menu bar full? That has happened to me too.

**Comment by fr-rose-steven** [comment]
No, there's enough space but the symbol wasn't there. I redownloaded and this seem to have fixed the issue somehow. No idea what is different. The version seems to be the same.

**Comment by fr-rose-steven** [comment]
I downloaded 4.17 and the issue reappeared...
